<?xml version="1.0"?>

<launch>
  <rosparam file="$(find decision_manager)/config/goal_manager.yaml" command="load"/>
  <node pkg="decision_manager" type="task_listener_loader" name="task_listener_loader" output="screen">
  </node>
</launch>

